kong-3.6.0-0.rockspec

```diff
@@ -565,6 +565,9 @@ build = {
     ["kong.plugins.ai-proxy.handler"] = "kong/plugins/ai-proxy/handler.lua",
     ["kong.plugins.ai-proxy.schema"] = "kong/plugins/ai-proxy/schema.lua",
 
+    ["kong.plugins.reconfiguration-completion.handler"] = "kong/plugins/reconfiguration-completion/handler.lua",
+    ["kong.plugins.reconfiguration-completion.schema"] = "kong/plugins/reconfiguration-completion/schema.lua",
+
```

Comment:
I think maybe we should not make this a bundled plugin yet, can we use the `pre-function` in the test to implement the logic for now as discussed? I do not know if we want to support this for customer's use yet.

Reply:
If we only want to use them in tests, then the best way to do it is by creating a "test-only" custom plugin in https://github.com/Kong/kong/tree/master/spec/fixtures/custom_plugins .

Hans however has told me that this functionality is actually quite interesting for Konnect themselves.

Reply:
Test fixtures do not exist in E2E tests, or older version of Kong Gateway that we test (e.g. 2.8), so serverless will be the easiest and safest way to implement this.

We can consider making this a feature, but not this late in the 3.6 cycle as some design work and downstream notification will be needed. Suggest we not consider it until after 3.6 is released.